### PR TITLE
clive 0.12.5 (new formula)

### PR DESCRIPTION
**Pull request by koki-develop**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Review by chenrui333 — APPROVED** [review]

File: Formula/clive.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+  depends_on "go" => :build
+  depends_on "ttyd"
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-s -w -X github.com/koki-develop/clive/cmd.version=v#{version}")
+  end
+
+  test do
+    system "#{bin}/clive", "init"
```
Comment (chenrui333) [inline Formula/clive.rb:16]:
```suggestion
    system bin/"clive", "init"
```

File: Formula/clive.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    assert_predicate testpath/"clive.yml", :exist?
+
+    system "#{bin}/clive", "validate"
```
Comment (chenrui333) [inline Formula/clive.rb:19]:
```suggestion
    system bin/"clive", "validate"
```

**Comment by chenrui333** [comment]
left some minor code style suggestions.

**Comment by koki-develop** [comment]
@chenrui333 
Thank you very much, I fixed it!

**Review by chenrui333 — APPROVED** [review]

**Pull request merged**